data/simulations/FAH-Mpro-monomer-no-water.yml

```diff
@@ -0,0 +1,75 @@
+type: md
+title: SARS-CoV-2 main protease (apo, monomer) Folding@home simulations
+description: >
+  This is a dataset containing 5688 trajectories at least 250 ns in length (2.6 ms in total) of SARS-CoV-2 main viral protease (Mpro/3CLpro) in its apo, monomeric form. 
+  Water (but not salt) has been stripped from the trajectories, frames are saved every 0.2 ns.
+  Simulations were initiated from PDB structure `6lu7`, chain A, after removing the inhibitor and structural waters.
+  
+  The dataset is organized by Folding@home project number (11743 and 11749) due to the F@h parallelization -- there are no differences in setups between the projects 
+  and there is no relation between the identically named files - all trajectories (`CLONE`s) are initialized with random velocities.
+
+  Chain A (i.e. a monomer of the protein, without the inhibitor or waters) was extracted from `6lu7` using PyMOL, and protonated and capped (ACE, NME) with Schrodinger's Maestro.
+  Simulations were performed in the NPT ensemble (310 K, 1 atm), in a cubic box with 1 nm padding, 150 mM NaCl, with hydrogen mass repartitioning (4 amu H mass), using `amber14SB` and `tip3p` forcefields.
+  OpenMM 7.4.1 was used. System was equilibrated for 5 ns using 2 fs timestep with default OpenMM Langevin integrator,
+  then for 1.25 ns using 4 fs timestep with OpenMMTools custom Langevin integrator using `V R O R V` splitting.
+  All Folding@home trajectories were then seeded with random velocities from this system.
+
+  The dataset is available through AWS Public Datasets and can be accessed through the command line interface (CLI) as follows:  
+    
+    1) Install the [AWS CLI](https://aws.amazon.com/cli/): 
+       ```bash
+       conda install -c conda-forge awscli
+       ```
+       or 
+       ```bash
+       pip install awscli
+       ```
+    2) To download the whole dataset (519 GB): 
+       ```bash
+       aws s3 sync --no-sign-request s3://fah-public-data-covid19-moonshot-dynamics/SARS-CoV-2_main_protease_monomer .
+       ```
+    3) To download subsets of the dataset appropriate query terms can be used, e.g.:  
+       the first trajectory of project 11743:
+       ```bash
+       aws s3 cp --no-sign-request s3://fah-public-data-covid19-moonshot-dynamics/SARS-CoV-2_main_protease_monomer/11743/run0-clone0.h5 .
```

Comment:
should be `sync`

Reply:
no, `sync` doesn't work for single files

Reply:
(leaving unresolved in case I'm wrong or there's any other comments)